spark-and-cassandra-on-centos/Scripts/Deploy-AzureResourceGroup.ps1

```diff
@@ -0,0 +1,103 @@
+﻿#Requires -Version 3.0
```

Comment:
Re: binaries (dlls and .exe), since we have no way to check/guarantee these are safe, please remove them for this PR and adapt this script to use the New-AzureRmResourceGroupDeployment cmdlet.